src/SecureHeaders.php

```diff
@@ -1246,30 +1189,32 @@ private function deconstructHeaderValue(
         return $attributes;
     }
 
-    private function validateHeaders()
+    private function validateHeaders(HeaderBag $headers)
```

Comment:
But it is? `$headers->forEachNamed('content-security-policy', ...)` - that is the special case.